### ISSUE TITLE
The results I test on the validation set are different from those provided in the paper?

### ISSUE THREAD

**Issue by rockywind** [TRAILab/CaDDN#30]
Thanks for your contribution!
The results I tested on the validation set is bellow:

![image](https://user-images.githubusercontent.com/14838781/120129489-a4724580-c1f6-11eb-9815-2d8a87644608.png)

but, the results provided in this paper is:
![image](https://user-images.githubusercontent.com/14838781/120129578-d2f02080-c1f6-11eb-91de-22987d902f76.png)

**Comment by codyreading** [comment]
Yup that is expected. Each time the model is trained the results will differ slightly. The model I trained for this code release is slightly different from the model I reported for the paper, as I had to do a refactor for this release that made my old model incompatible.

**Comment by rockywind** [comment]
Yes,thank you very much!

**Comment by SirJamie** [comment]
@rockywind  hi, may I ask what is your training config? 😊
